consensus-peer/config.proto

```diff
@@ -14,10 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package helper
+syntax = "proto3";
 
-import "testing"
+package main;
 
-func TestHandler(t *testing.T) {
-	t.Skip("Handler functions already tested in other consensus components")
-}
+import "github.com/hyperledger/fabric/consensus/pbft/config.proto";
+
+message config {
+        pbft.batch_config consensus = 1;
+        map<string, bytes> peers = 2;
+};
```

Comment:
I agree.  I'd say use a hash of the genesis configuration as identifier for the network.